src/app/malaria-101/activity-3/activity-3.component.ts

```diff
@@ -91,11 +106,10 @@ export class OddOneOutComponent implements OnInit {
         } else {
             this._sharedData.customErrorAlert(this.alerts.activityFailMsg, this.alerts.activityFailTitle);
         }
-        this._obs = observableInterval(1000).pipe(
+        this._obs = observableInterval(10).pipe(
```

Comment:
Why did you change this?

Reply:
Here, 1000 was the time delay between displaying the next question, when the user has answered the previous one. So, this being not a smooth experience, I reduced the delay to 10ms.